backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/ContextInitializer.scala

```diff
@@ -66,6 +66,8 @@ class ContextInitializer extends ContextApi {
       new SharedLibraryLoaderCentos8
     } else if (systemName.contains("Anolis") && systemVersion.startsWith("7")) {
       new SharedLibraryLoaderCentos7
+    } else if (system.contains("tencentos") && system.contains("3.2")) {
+      new SharedLibraryLoaderCentos8
```

Comment:
Nit:
Please include the new supported OS in the below exception message (Line#74).